### DISCUSSION TITLE
Report abuse comment

### DISCUSSION THREAD

**Discussion by Constey** [community/community#136842] [Code Security]
### Select Topic Area

General

### Body

I cannot report the comment directly over the comment, because there is no option. But seems to be a virus:
https://github.com/AtlassianPS/ConfluencePS/issues/214#issuecomment-2309804631
![image](https://github.com/user-attachments/assets/dafea06b-2f36-4f82-a94a-6e231af30b7e)
can anyone remove this?

**Comment by MitchBomcanhao** [comment]
these are snowballing across many repos using multiple user accounts.
you might need to act fast to stop this...
searching for part of that message, you can find thousands of matches, and they keep piling on
